.github/ISSUE_TEMPLATE/config.yaml

```
@@ -0,0 +1,9 @@
+---
+blank_issues_enabled: true
+contact_links:
+  - name: Ask a question
+    url: https://github.com/containers/buildah/discussions/new
+    about: Ask a question about Buildah
+  - name: If your Podman issue is not related to podman build, please report it here
```

I think the `not` should be dropped here, or perhaps I'm just having a brain-fart.
```suggestion
  - name: If your Podman issue is related to podman build, please report it here
```

---

We want non-`podman build` issues to be filed directly in the podman repository.

---

I think you current suggestion is incorrect. I want this to read as, "if your issue is a general Podman issue, not related to `podman build`, then open it in the Podman repo" - If you think the current text is confusing could you please suggest an alternative? Thank you!

---

I think your example in your last comment makes sense.  How's about:

If your issue is a general Podman issue unrelated to `podman build`, please open an [issue](https://github.com/containers/podman/issues) in the Podman repository.  If the issue is with the `podman build` command, please report it here.

---

Thanks Tom, Done!